Sources_v3/StreamChat/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -185,10 +185,17 @@ class ChannelController_Tests: StressTestCase {
     }
 
     func test_channelControllerForNewChannel_throwsError_ifCurrentUserDoesNotExist() throws {
+        let clientWithoutCurrentUser = ChatClient(
+            config: .init(apiKeyString: .unique),
+            tokenProvider: .closure {
+                $1(.failure(TestError()))
+            }
```

Comment:
Maybe we can introduce some kind of `.invalid` helper for tests?

Reply:
introduced in 381b4b8f2dd4633078258599584e7a0e7758fb76